### PR TITLE
[doc][yba] 2024.1 OIDC group mapping

### PR DESCRIPTION
**Pull request by ddhodge**
YBA OIDC group mapping
DOC-329

@netlify /preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md [hunk 1]
```diff
@@ -63,10 +85,20 @@ You configure OIDC as follows:
...
 
+        If you have an airgapped installation, where YBA cannot access the Discovery URL, provide the OIDC configuration for the identity provider directly.
+
+        To do this, click **Configure OIDC Provider Metadata** and paste the OIDC configuration document from your identity provider (in JSON format) into the field.
+
     - In the **Scope** field, enter your identity provider OIDC scope that is allowed to be requested. This field accepts a space-separated list of values. If left blank, all scopes will be considered.
     - In the **Email Attribute** field, enter the OIDC scope containing the user email identifier. This field accepts a case-sensitive custom configuration. Typically, this field is left blank.
-    - If you have an airgapped installation, where YBA cannot access the Discovery URL, provide the OIDC configuration for the identity provider directly.
+    - In the **Refresh Token URL** field, enter the URL of the refresh token.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:94]: @asharma-yb can you provide more info on this option

**Comment by ddhodge** [comment]
@asharma-yb can you review and/or approve this

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md [hunk 2]
```diff
@@ -43,6 +43,28 @@ To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super A
 - For information on configuring a YugabyteDB Anywhere universe to use OIDC-based authentication using Azure AD as the IdP, refer to [OIDC authentication with Azure AD](../../security/authentication/oidc-authentication-aad/).
 - For information on how to add users, see [Create, modify, and delete users](../anywhere-rbac/#create-modify-and-delete-users). The email ID that you enter in the **Add User** dialog must be registered with the identity provider, and the role must reflect the user's role on YugabyteDB Anywhere.
 
+## Use OIDC groups with YBA roles
+
+If your OIDC provider is configured with user groups, you can map the groups to [YBA roles](../anywhere-rbac/). Users who are members of these groups can then sign in to YBA without needing to be added to YBA first. Users who are members of multiple groups are assigned the most privileged role.
```
Comment (asharma-yb) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/oidc-authentication.md:48]: >YBA roles
We should mention somewhere that we only support built in roles for now and that custom roles will be supported in future releases.

**Review by asharma-yb — APPROVED** [review]
One minor comment.

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request closed without merging**